components/change_url_modal/change_url_modal.tsx

```diff
@@ -12,73 +11,82 @@ import Constants from 'utils/constants';
 import {getShortenedURL, cleanUpUrlable} from 'utils/url';
 import {t} from 'utils/i18n';
 
-export default class ChangeURLModal extends React.PureComponent {
-    static propTypes = {
+export type Props = {
 
-        /**
-        * Set whether to show the modal or not
-        */
-        show: PropTypes.bool.isRequired,
+    /**
+     * Set whether to show the modal or not
+     */
+    show: boolean;
 
-        /**
-        * Set to change the title of the modal
-        */
-        title: PropTypes.node,
+    /**
+     * Set to change the title of the modal
+     */
+    title: JSX.Element | string;
```

Comment:
The equivalent for `PropTypes.node` in typescript is actually `React.ReactNode`.
As this not required, could we add a conditional `?` to it? Like this
```suggestion
    title?: React.ReactNode;
```

Reply:
Fixed!